server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/delivery/SieveMailet.java

```diff
@@ -220,4 +312,190 @@ public void post(String url, MimeMessage mail) throws MessagingException {
         }
     }
 
+
+    /**
+     * Delivers a mail to a local mailbox.
+     *
+     * @param mail
+     *            the mail being processed
+     *
+     * @throws MessagingException
+     *             if an error occurs while storing the mail
+     */
+    @SuppressWarnings("unchecked")
+    @Override
+    public void service(Mail mail) throws MessagingException {
+        Collection<MailAddress> recipients = mail.getRecipients();
+        Collection<MailAddress> errors = new Vector<MailAddress>();
```

Comment:
I just keep underlying Collection during refactoring.

Do you have any rationnal reason to not like vectors ? Should I change ?